ckanext/canada/tables/travelq.yaml

```
@@ -187,37 +187,189 @@ resources:
   - datastore_id: destination_en
     datastore_type: text
     label:
-      en: Places visited (English)
+      en: Place visited (English)
       fr: Endroits visités (anglais)
     description:
-      en: To include names of all places visited during the trip in question, in English
+      en: To include name of the place visited during the trip in question, in English.
```

```suggestion
      en: To include name of the first place visited during the trip in question, in English.
```
?

Also really unfortunate to see these fields collected in French and English. How often will they be different? Wouldn't using a geocoder on this data to normalize and present both languages be much better?

---

Yeah it is really stupid to collect these fields in both languages. I doubt that many would be different in French. Idk how we would do the geo stuff though, maybe an improvement in the future as the timeline for this is tight and not sure which library would be best and the network limitations for any API stuff we would possibly need.

So I guess we will just collect them for now...